db/migrate/20190408004639_add_avail_sun_am_to_volunteers.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvailSunAmToVolunteers < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
These 14 migration files should be organized as onefile .  We can make this change as no merge of them has been done.  What you want is 14 rows saying add_column :volunteers, :avail_sun_am, :string followied by add_column :volunteers, :avail_sun_pm, :string and so on for all 14, all within a single change.

Reply:
Completed edit